src/rebar_app_discover.erl

```diff
@@ -437,8 +442,8 @@ to_atom(Bin) ->
 find_config_src(AppDir, Default) ->
     case rebar_config:consult(AppDir) of
         [] ->
-            Default;
+            {[], Default};
         Terms ->
             %% TODO: handle profiles I guess, but we don't have that info
-            proplists:get_value(src_dirs, Terms, Default)
+            {Terms, proplists:get_value(src_dirs, Terms, Default)}
```

Comment:
unless I'm wrong, find_config_src meant the 'source config' not 'the config and the src dirs' ?

Reply:
Na, it is a function that returns src dirs. Before it just threw away the config though. So we'd be reading in the config, checking for `src_dirs` only and then later having to re-read it in and evail the .script again.

Reply:
Got it